src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_help.py

```diff
@@ -12,9 +12,94 @@
     short-summary: commands for managing your Azure web apps, app service plans, etc
 """
 
-helps['appservice plan'] = """
+helps['appservice web config'] = """
     type: group
-    short-summary: commands for managing your app service plans
+    short-summary: commands to configure the web
```

Comment:
Commands to configure your webapps?  "commands to configure the web" sounds really REALLY powerful!